gslib/commands/signurl.py

```diff
@@ -128,7 +129,12 @@
                the duration the link remains valid is the sum of all the
                duration options.
 
-               The max duration allowed is 7d.
+               The max duration allowed is 7 days when ``private-key-file``
+               is used.
+
+               The max duration allowed is 12 hours when -u option is used.
+               This limitation exists because the system managed key used to
+               sign the url is guaranteed to remain valid for at least 12 hours.
```

Comment:
Should this be "at most 12 hours"?

Reply:
It should be "at least". Basically, the key is guaranteed to remain valid for at least 12 hours. After 12 hours, the key may or may not be valid. Happy to rephrase this if it sounded confusing and if there are other suggestion :)

Reply:
Possible wordsmithing: "This limitation exists because the system-managed key used to sign the url is rotated regularly, with 12 hours being the lower limit of time that the system-managed key is valid"

Reply:
Talked to @starsandskies offline and we decided on an alternate wording for this.